community/security/src/main/java/org/neo4j/server/security/auth/User.java

```diff
@@ -94,6 +104,7 @@ public boolean equals( Object o )
     public int hashCode()
     {
         int result = name != null ? name.hashCode() : 0;
+        result = 31 * result + ( group != null ? group.hashCode() : 0);
```

Comment:
Or is this only used to decide if the line in the auth file should be updated and the current location of a group field in that file means we need to do this?